packages/badge/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Products } from '@hashicorp/platform-product-meta'
+import classnames from 'classnames'
+import s from './style.module.css'
+
+interface BadgeProps {
+  children: string
+  theme?: Products | 'neutral' | 'white' | 'black'
+  variant?: 'primary' | 'secondary'
+}
+
+export default function Badge({
+  children,
+  variant = 'primary',
+  theme = 'neutral',
+}: BadgeProps) {
+  return (
+    <span className={classnames(s.badge, s[theme], s[variant])}>
```

Comment:
Did we look at using the product meta styles for this? https://github.com/hashicorp/web-platform-packages/tree/main/packages/product-meta

Reply:
I did, but the theme support more than our product colors. We'd be loading unnecessary variable declarations that are not documented to use within the Figma component implementation.

Reply:
The product meta theme classes are more than likely already in the document though, so it shouldn't be anything net-new. We could  use a combination of the styles from product meta as well as ones which are for the unique themes that `Badge` supports. Which unnecessary variable declarations are we worried about?

Reply:
@BRKalow Here is an update making use of productStyles https://github.com/hashicorp/react-components/pull/512/commits/282709c8e36a8166ca5df4627c8f8d4fa4689187

Reply:
![oh snap](https://i.giphy.com/media/rVVFWyTINqG7C/giphy.webp)